webapp/channels/src/components/root/root.tsx

```diff
@@ -361,7 +361,9 @@ export default class Root extends React.PureComponent<Props, State> {
         }
 
         const myTeams = getMyTeams(storeState);
-        if (myTeams.length > 0) {
```

Comment:
@pvev in the case of the rainforest server, they did have a teams setup but this didn't work becuase the admin may not have been part of the team? 

What is the behavior if the flag is off and there is no teams?

Reply:
with the current implementation if there are no teams and the flag is off, the redirection will go to /preparing-workspace in case of first admin and hasn't completed already the setup or to the default team in any other user profile or admin that has already completed the set-up

Reply:
with the current implementation if there are no teams and the flag is off, the redirection will go to /preparing-workspace in case of first admin and hasn't completed already the setup or to the default team in any other user profile or admin that has already completed the set-up